dashel/dashel-common.cpp

```diff
@@ -59,83 +59,67 @@
 namespace Dashel
 {
 	using namespace std;
-	
+
 	// frome dashe-private.h
-	ExpandableBuffer::ExpandableBuffer(size_t size) :
-		_data((unsigned char*)malloc(size)),
-		_size(size),
-		_pos(0)
-	{
-	}
-	
+	ExpandableBuffer::ExpandableBuffer(size_t size) : _data((unsigned char*)malloc(size)), _size(size), _pos(0) {}
+
 	ExpandableBuffer::~ExpandableBuffer()
 	{
 		free(_data);
 	}
-	
+
 	void ExpandableBuffer::clear()
 	{
 		_pos = 0;
 	}
-	
+
 	void ExpandableBuffer::add(const void* data, const size_t size)
 	{
 		if (_pos + size > _size)
 		{
 			_size = max(_size * 2, _size + size);
 			_data = (unsigned char*)realloc(_data, _size);
 		}
-		memcpy(_data + _pos, (unsigned char *)data, size);
+		memcpy(_data + _pos, (unsigned char*)data, size);
 		_pos += size;
 	}
-	
+
 	// to be removed when we switch to C++11
 	string _to_string(int se)
 	{
 		ostringstream ostr;
 		ostr << se;
 		return ostr.str();
 	}
-	
+
 	// frome dashel.h
-	DashelException::DashelException(Source s, int se, const char *reason, Stream* stream) :
+	DashelException::DashelException(Source s, int se, const char* reason, Stream* stream) :
 		std::runtime_error(sourceToString(s) + " (" + _to_string(se) + "): " + reason),
 		source(s),
 		sysError(se),
 		stream(stream)
 	{
-	
 	}
-	
+
 	string DashelException::sourceToString(Source s)
 	{
-		const char* const sourceNames[] =
-		{
-			"Unknown cause",
-			"Synchronisation error",
-			"Invalid target",
-			"Invalid operation",
-			"Connection lost",
-			"I/O error",
-			"Connection failed",
-			"Enumeration error",
-			"Previous incoming data not read"
-		};
-		const size_t arrayLength(sizeof(sourceNames)/sizeof(const char*));
+		const char* const sourceNames[] = {
```

Comment:
This is much less readable than the previous version.

Reply:
I know, the tool can't deal with that

Reply:
Then we should manually revert that, readability is key :)